packages/console/test/throws-and-logs.js

```diff
@@ -0,0 +1,139 @@
+import { cycleTolerantStringify } from '@agoric/assert';
```

Comment:
I’m skeptical about `cycleTolerantStringify` being part of the committed public API of the `assert` module. I would go so far as to duplicate it rather than create a dependency. Or, just avoid using it.